app/services/releasecop_service.rb

```diff
@@ -16,13 +16,15 @@ def initialize(project)
   end
 
   def perform_comparison
-    Dir.mktmpdir(['releasecop', project.name]) do |dir|
-      checker = Releasecop::Checker.new(
-        project.name,
-        project.stages.order(position: :asc).map { |s| build_manifest_item(s) },
-        dir
-      )
-      ResultWrapper.new(checker) # build comparisons
+    env_dir = Horizon.config[:perform_comparison_workdir]
```

Comment:
Minor: I'd call this variable and config `working_dir` for consistency and to match [releasecop](https://github.com/joeyAghion/releasecop/blob/bc9c108f1c4ff33fefe75ffb0c44f5ac8822594d/lib/releasecop/checker.rb#L5).

Reply:
sounds fine to me. updated.